src/cf.data.pre

```diff
@@ -2894,16 +2903,22 @@ DOC_START
 			To control SSLv3 use the options= parameter.
 			Supported Values: 1.0 (default), 1.1, 1.2
 
-	options=...	Specify various TLS/SSL implementation options.
+	options=	Library specific options.
```

Comment:
If possible, please remove this documentation duplication by simply referring to `tls-options` here. For example:
```suggestion
	options=	See http_port tls-options.
```